app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java

```diff
@@ -48,6 +61,7 @@
     protected View rootInstructionLayout;
     protected ImageButton customButton;
     protected TextView customButtonText;
+    protected Button remindMeLaterButton;
```

Comment:
Can this move with the handling code to CrfSkipInstructionStepLayout?

Reply:
Moved